src/Microsoft.OData.Core/JsonLight/JsonLightConstants.cs

```diff
@@ -98,7 +101,13 @@ internal static class JsonLightConstants
         /// <summary>The simplified OData Context property name.</summary>
         internal const string SimplifiedODataContextPropertyName = "@context";

```

Comment:
can we remove the "OData" in SimplifiedODataXXXX?

Reply:
Well, we already have things like ODataIdPropertyName, so it's kinda nice to have a consistent naming pattern to disambiguate the strings used for simplified options.  We don't have a non-simplified form for ODataRemoved, but I'll add OData to the name for consistency.

---
In reply to: [150045071](https://github.com/OData/odata.net/pull/989#discussion_r150045071) [](ancestors = 150045071)